src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -442,4 +446,50 @@ private boolean checkAcceptedBody(String accept) {
         }
         return false;
     }
+
+    /**
+     * Build the request URI for the future consumer requests
+     *
+     * @param routingContext context of the current HTTP request
+     * @return the request URI for the future consumer requests
+     */
+    private String buildRequestUri(RoutingContext routingContext) {
+        String baseUri = routingContext.request().absoluteURI();
+
+        String xForwarded = routingContext.request().getHeader("x-forwarded");
+        if (xForwarded != null && !xForwarded.isEmpty()) {
+            Matcher hostMatcher = hostPattern.matcher(xForwarded);
+            Matcher protoMatcher = protoPattern.matcher(xForwarded);
+            if (hostMatcher.find() && !hostMatcher.group(1).isEmpty() &&
```

Comment:
It can't be empty because you used `+` not `*`.

Reply:
good spot ;)